kahuna/public/js/components/gr-collections-panel/gr-collections-panel.css

```diff
@@ -0,0 +1,70 @@
+gr-nodes, gr-node {
+    display: block;
+}
+
+.panel__left {
+    float: left;
+    position: fixed;
+    bottom: 0;
+    z-index: 1;
+    transition: left 0.1s;
+}
+
+.panel__left--hidden {
+    left: -290px;
+    transition: left 0.1s;
+}
+
+.node__action {
+    display: none;
+}
+
+.node__children {
+    padding-left: 10px;
+}
+
+.node__info.drag-over .node__name {
+    background-color: #333;
+}
+
+.node__info:hover .node__marker {
+    display: block;
+}
+
+.node__info:hover .node__action {
+    display: block;
+}
+
+.node__marker {
+    position: absolute;
+    background: hotpink;
```

Comment:
> Nice! Does it fit the Grid's theme though?

Although I'm pro hotpink, I think not. @alastairjardine: the blue used elsewhere e.g. 'upload' button?